msa-rest/src/main/java/cz/muni/fi/pa165/msa/rest/controllers/UserController.java

```diff
@@ -0,0 +1,50 @@
+package cz.muni.fi.pa165.msa.rest.controllers;
+
+
+import cz.muni.fi.pa165.msa.dto.UserDTO;
+import cz.muni.fi.pa165.msa.facade.UserFacade;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping
```

Comment:
Please add here actual mapping. String "/products" should be enough for now.

Reply:
@MiroslavLuckai why `/products`?